sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/QueueReceiverClient.cs

```diff
@@ -42,8 +42,30 @@ public QueueReceiverClient(string connectionString)
         {
         }
 
+
         /// <summary>
-        ///   Initializes a new instance of the <see cref="ServiceBusReceiverClient"/> class.
+        ///   Initializes a new instance of the <see cref="ServiceBusReceiver"/> class.
+        /// </summary>
+        ///
+        /// <param name="connectionString">The connection string to use for connecting to the Service Bus namespace; it is expected that the Service Bus entity name and the shared key properties are contained in this connection string.</param>
+        /// <param name="sessionOptions"></param>
```

Comment:
nit:  missing text.    Leaving just this one, but all the doc comments seem to have parameters not filled in.

Reply:
I wanted to do a separate pass on documentation before any previews to avoid everything is solid. I have issues tracking this.

Reply:
Then I'll stop nitpicking doc comment issues.